app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -60,12 +60,12 @@ def start
     begin
       kubernetes_client.create_pod(pod)
     rescue SocketError, KubeException => e
-      msg = "pod creation for #{pod_full_name} failed: #{e}"
-      _log.info(msg)
-      return queue_signal(:abort_job, msg, "error")
+      return queue_abort("pod creation for #{pod_full_name} failed: #{e}")
```

Comment:
Or if you rely on re-raising to log backtrace, like in #8747, you probably also want `raise`s in this PR too?